resources/postgres/resources_linux.go

```diff
@@ -1,5 +1,5 @@
-//go:build linux
-// +build linux
+//go:build !alpine
```

Comment:
I would still prefer to use explicit tag, if linux is a reserved tag, we can say standard-linux as opposed to alpine-linux?

Reply:
In a somewhat related discussion, the file suffix will make cross-compilation easier without duplicating the GOOS / ARCH flags in the tags part. Also it will make life easier if we have both GOOS and ARCH cross compilation later. More importantly, if we use explicit tag, we have to find other ways to name the suffix of the file or we have to use file suffix for platforms.

https://groups.google.com/g/golang-nuts/c/LD7VuMhQhPs

Reply:
If not using build tag, we will include both linux and darwin into the binary.

Binary is a concern for both docker and non-docker distribution.

Reply:
If we use GOOS as file suffix, only the file asssociated with the platform will be included in the build. If not, we will receive re-declaration of variable resources on build. I also tested on a EC2 linux machine, build_docker.sh will produce an image of size 75.9M which should only include the alpine linux dist.

[centos@ip-172-31-27-6 bytebase]$ docker images
REPOSITORY                   TAG                IMAGE ID      CREATED         SIZE
localhost/bytebase/bytebase  latest             557d13a5a7ae  9 seconds ago   75.9 MB

Reply:
I see. Then probably to add a comment in resources/readme

Reply:
And to explain why we "use alpine build tags otherwise use file suffix build tags since linux is a reserved tag"

Why alpine is special here.

Reply:
Done.